mysql-test/suite/join/join_natural.test

```diff
@@ -0,0 +1,46 @@
+--echo #
+--echo # Bug #XXXXXX: INET_ATON() returns signed, not unsigned
```

Comment:
MDEV-24931 instead of Bug.

Using an existing test file like #2520 is referred.